### PR TITLE
update script.py and integration.py to get_remote_file

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
Ready

## Description
`origin` should not be part of the github url, it is only used when running git commands.

see first and last build in this branch: https://circleci.com/gh/demisto/content/tree/TestRemoteFile

## Screenshots
image

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

File: Tests/test_utils.py [hunk 1]
```diff
@@ -60,12 +63,18 @@ def run_command(command, is_silenced=True, exit_on_error=True):
...
-    res = requests.get(github_path, verify=False)
-    if res.status_code != 200:
+    try:
+        res = requests.get(github_path, verify=False)
+        res.raise_for_status()
+    except Exception as exc:
         print_warning('Could not find the old entity file under "{}".\n'
                       'please make sure that you did not break backward compatibility. '
-                      'Reason: {}'.format(github_path, res.reason))
+                      'Reason: {}'.format(github_path, exc))
```
Comment (Itay4) [inline Tests/test_utils.py]:
```suggestion
                      'Reason: {}'.format(github_path, str(exc)))
```

File: Tests/scripts/hook_validations/integration.py [hunk 2]
```diff
@@ -141,15 +124,12 @@ def is_default_arguments(self):
             command_name = command.get('name')
             for arg in command.get('arguments', []):
                 arg_name = arg.get('name')
-                if ((command_name == 'file' and arg_name == 'file')
-                        or (command_name == 'email' and arg_name == 'email')
-                        or (command_name == 'domain' and arg_name == 'domain')
-                        or (command_name == 'url' and arg_name == 'url')
-                        or (command_name == 'ip' and arg_name == 'ip')):
+                if command_name in {'file', 'email', 'domain', 'url', 'ip'} and arg_name == command_name:
```
Comment (Itay4) [inline Tests/scripts/hook_validations/integration.py]:
```suggestion
                if command_name in ('file', 'email', 'domain', 'url', 'ip') and arg_name == command_name:
```

Comment (yaakovi) [reply Tests/scripts/hook_validations/integration.py]: why tuple and not a set?

File: Tests/test_utils.py (hunk 1 above)
Comment (yaakovi) [reply Tests/test_utils.py]: no need:
image

**Pull request merged**